lxd/auth/oidc/oidc.go

```diff
@@ -304,14 +311,32 @@ func getAccessTokenVerifier(issuer string) (op.AccessTokenVerifier, error) {
 }
 
 // NewVerifier returns a Verifier.
-func NewVerifier(issuer string, clientid string, audience string) (*Verifier, error) {
-	cookieKey, err := uuid.New().MarshalBinary()
+func NewVerifier(issuer string, clientid string, audience string, clusterCert *shared.CertInfo) (*Verifier, error) {
+	// Derive secret keys from the server cert (standalone) or network cert (cluster) to ensure all members use the
```

Comment:
not "cert" but "key". The cert is public.

Reply:
Whoops yep will fix now

Reply:
Done.